source/keyboardHandler.py

```diff
@@ -494,6 +494,66 @@ def __init__(self, modifiers, vkCode, scanCode, isExtended):
 		self.isExtended = isExtended
 		super(KeyboardInputGesture, self).__init__()
 
+	def _get_character(self):
+		threadID = api.getFocusObject().windowThreadID
+		keyboardLayout = ctypes.windll.user32.GetKeyboardLayout(threadID)
+		buffer = ctypes.create_unicode_buffer(5)
+		states = (ctypes.c_byte * 256)()
+		modifierList = []
+		valid = True
+		for i in self.modifiers:
+			if i[0] not in self.NORMAL_MODIFIER_KEYS.values():
+				modifier = self.NORMAL_MODIFIER_KEYS.get(i[0])
+			else:
+				modifier = i[0]
+			if modifier:
+				modifierList.append(modifier)
+		for i in range(256):
+			if i in modifierList:
+				states[
+					i
+				] = (
+					-128
+				)  # We tell ToUnicodeEx that the modifier is down, even tho it isn't according to Windows
+			else:
+				states[i] = ctypes.windll.user32.GetKeyState(i)
+		res = ctypes.windll.user32.ToUnicodeEx(
+			self.vkCode,
+			self.scanCode,
+			states,
+			buffer,
+			ctypes.sizeof(buffer),
+			0x0,
```

Comment:
Should this be
```suggestion
			0x04
```
See a similar call in `internal_keyDownEvent`